index.html

```diff
@@ -296,8 +296,11 @@ <h2 id="setup">Setup</h2>
 
 <p>
   To participate in a Software Carpentry workshop, you will need
-  access to the software described below. In addition, you will
-  need an up-to-date web browser.
+  access to the software described below as well as an up-to-date web browser. 
+  You will also need access to the 
+  <a href = "https://swcarpentry.github.io//shell-novice/shell-novice-data.zip">shell dataset</a>
+  and the <a href = "https://github.com/swcarpentry/r-novice-gapminder/blob/gh-pages/data/gapminder-FiveYearData.csv">
+  gapminder dataset</a>.
```

Comment:
Last time we didn't really need the gapminder dataset because it was installed as an R package; is it needed this time?